common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,124 @@
+"""OLI analytics service event tracker backend."""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from requests_oauthlib import OAuth1Session
+
+from django.contrib.auth.models import User
+
+from student.models import anonymous_id_for_user
+from track.backends import BaseBackend
+
+
+LOG = logging.getLogger(__name__)
+
+
+class OLIAnalyticsBackend(BaseBackend):
+    """
+    Transmit events to the OLI analytics service
+    """
+    def __init__(self, **kwargs):
+        super(OLIAnalyticsBackend, self).__init__(**kwargs)
+
+        # the oli analytics service endpoint will be at
+        # "{}{}".format(oli_analytics_service_url, path)
+        self.oli_analytics_service_url = kwargs.get('oli_analytics_service_url', '')
```

Comment:
Can we just call this member `url`?
The `oli_analytics_service_` prefix is superfluous as that information
is already implied by the class name itself, `OLIAnalytics`.

This helps to avoid the Java-like over-verbosity of statements like
`OLIAnalyticsBackend.oli_analytics_service_url` instead of the much
simpler `OLIAnalyticsBackend.url`.

The shorter variant is more concise, without any loss of information.
Additionally, it follows the convention used for the other instance
members (`path`, `key`, `secret`, `oauth`, etc.), none of which carry
this additional prefix.